paper.tex

```diff
@@ -69,6 +69,8 @@ \subsection{Detector models}
 
 \subsubsection{Image detector models}
 
+The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 finetuned on the AI or Not dataset. swin-tiny-patch4-window7-224 by microsoft is a new vision Transformer \cite{https://arxiv.org/pdf/2103.14030.pdf}.
```

Comment:
Maybe write a little bit more about swin-tiny-patch4-window7-224

Reply:
Or maybe after merging the resnet18 detector write something like "similiar to the aforementioned detector, this one is also fintuned on the AI or not dataset .... "

Because if we don't do it like that the whole text in itself will sound very choppy

Reply:
yea idk i would check for fluent text reading after we merged everything and not before couse i think we have to change it later anyway again.

Reply:
Yes but this garantees that we do at least twice the work, otherwise if we do it halfway decent the first time, we could prevent the possibility of doing it twice
But I'll get your point